### ISSUE TITLE
Invalid main and types path in package.json (Error: Cannot find module)

### ISSUE THREAD

**Issue by AlexanderOMara** [jet2jet/resedit-js#2]
The `main` and `types` paths `package.json` are incorrect (at-least in the version published to NPM).

They are set to:

```json
"main": "./dist/index.js",
"types": "./dist/index.d.ts",
```

But those files are are actually at:

```
./dist/src/main/index.js
./dist/src/main/index.d.ts
```

When attempting to load the module I get this error.

> Error: Cannot find module '.../node_modules/resedit/dist/index.js'. Please verify that the package.json has a valid "main" entry

I can still use it if I use a deep-import, but it would be nice if this were fixed properly.

```js
require('resedit/dist/src/main')
```

PS. This is a pretty cool module. With a little scripting I think this can be used as a platform-independent replacement for rcedit.

**Comment by jet2jet** [comment]
Thanks for creating this issue and a PR! I'll verify and merge the PR as soon as possible.

**Comment by jet2jet** [comment]
(I published as [v0.2.1](https://github.com/jet2jet/resedit-js/releases/tag/v0.2.1).)
